### PR TITLE
Add a new validator for token usage

### PR DESCRIPTION
**Pull request by vad33r**
This PR adds a new file that contains a class, this class inherits from EdxOAuth2Validator.

### CODE REVIEW

**Comment by vad33r** [comment]
im not sure where would be the correct place to put this file, so... any idea? @andrey-canon @magajh

**Review by andrey-canon — CHANGES_REQUESTED** [review]

File: eox_tenant/validators.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+"""
+Class that overwrite the default django-oauth-toolkit behavior
+"""
+from openedx.core.djangoapps.oauth_dispatch.dot_overrides.validators import EdxOAuth2Validator
```
[the remark this answers is not in the document]
Comment (vad33r) [reply eox_tenant/validators.py:4]: OK

File: eox_tenant/validators.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"""
+Class that overwrite the default django-oauth-toolkit behavior
```
[the remark this answers is not in the document]
Comment (vad33r) [reply eox_tenant/validators.py]: solved

File: eox_tenant/validators.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+"""
+from openedx.core.djangoapps.oauth_dispatch.dot_overrides.validators import EdxOAuth2Validator
+
+
+class EoxTenantOAuth2Validator(EdxOAuth2Validator):
+    """
+    Class that validate the token usage:
```
[the remark this answers is not in the document]
Comment (vad33r) [reply eox_tenant/validators.py]: solved

File: eox_tenant/validators.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+"""
+This overwrites the EdxOAuth2Validator behavior
+"""
+from openedx.core.djangoapps.oauth_dispatch.dot_overrides.validators import EdxOAuth2Validator
+
+
+class EoxTenantOAuth2Validator(EdxOAuth2Validator):
+    """
+    Class that validates the token usage:
+        * The token only can be used in the site it was created
```
Comment (magajh) [inline eox_tenant/validators.py:10]: The token **can only be used**

Comment (vad33r) [reply eox_tenant/validators.py]: ty, solved

File: eox_tenant/validators.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (vad33r) [reply eox_tenant/validators.py]: backend created

File: eox_tenant/edxapp_wrapper/backends/validators_j_v1.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+""" Backend abstraction. """
+from openedx.core.djangoapps.oauth_dispatch.dot_overrides.validators import (  # pylint: disable=import-error
+    EdxOAuth2Validator,
+)
+
+def get_edx_oauth2_validator_class():
+    """ Backend to get EdxOAuth2Validator class
+    """
+    return EdxOAuth2Validator
```
Comment (magajh) [inline eox_tenant/edxapp_wrapper/backends/validators_j_v1.py:9]: Missing newline

Comment (vad33r) [reply eox_tenant/edxapp_wrapper/backends/validators_j_v1.py]: ty, solved

**Review by andrey-canon — APPROVED** [review]

**Pull request merged**